benchmarks/entropy_adiabat/plugins/entropy_model.cc

```diff
@@ -164,16 +286,24 @@ namespace aspect
           prm.declare_entry ("Material file name", "material_table.txt",
                              Patterns::List (Patterns::Anything()),
                              "The file name of the material data.");
-          prm.declare_entry ("Lateral viscosity prefactor", "0.0",
+          prm.declare_entry ("Reference viscosity", "1e22",
                              Patterns::Double(0),
-                             "The lateral viscosity prefactor used for computing the temperature"
-                             "dependence of viscosity. This corresponds to H/nR (where H is the "
-                             "activation enthalpy, n is the stress exponent and R is the gas "
-                             "constant) in an Arrhenius-type viscosity law, and is modeled after "
-                             "the law for lateral viscosity variations given in Steinberger and "
-                             "Calderwood, 2006. "
+                             "The viscosity that is used in this model. "
                              "\n\n"
-                             "Units: $1/K$");
+                             "Units: \\si{\\pascal\\second}");
+          // prm.declare_entry ("Lateral viscosity prefactor", "0.0",
+          //                    Patterns::Double(0),
+          //                    "The lateral viscosity prefactor used for computing the temperature"
+          //                    "dependence of viscosity. This corresponds to H/nR (where H is the "
+          //                    "activation enthalpy, n is the stress exponent and R is the gas "
+          //                    "constant) in an Arrhenius-type viscosity law, and is modeled after "
+          //                    "the law for lateral viscosity variations given in Steinberger and "
+          //                    "Calderwood, 2006. "
+          //                    "\n\n"
+          //                    "Units: $1/K$");
```

Comment:
Remove all commented source code. We never merge commented code, because it will be impossible in the future to decide if this is is outdated code, or if it is code to be fixed and included in the future.